DOCUMENTATION.md

```diff
@@ -259,35 +317,46 @@ This object is returned by the method `get()` of the ApplicationClient class. He
 * `register_on_join_access_key`: **string**
 
 ## HandlerClient
+
 The class constructor can be called following this scheme:
+
 ```python
-HandlerClient(app_id, access_key_or_token, [discovery_address], [certificate])
+HandlerClient(app_id, access_key_or_token, discovery_address="discovery.thethings.network:1900", certificate="")
 ```
-- `app_id`: **string**  this the name given to the application when it was created.
-![Screenshot of the console with app section](./images/app-console.png?raw=true)
-- `app_access_key`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**. The key needs the `settings`authorization.
-- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered. Default to `None`. If the `discovery_address` is not set, the client will use the address of The Things Network's discovery server: `discovery.thethings.network:1900`.
-- `certificate`: **string**  this is the certificate used to connect in a secure way to the discovery server. Default to `None`. If a discovery_address was provided, not providing the certificate will raise an exception.
+
+* `app_id`: **string** this the name given to the application when it was created.
+  ![Screenshot of the console with app section](./images/app-console.png?raw=true)
+* `app_access_key`: **string** this can be found at the bottom of the application page under **ACCESS KEYS**. The key needs the `settings`authorization.
+* `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered, in the `{hostname}:{port}` format.
+* `certificate`: **string** this is the path to the certificate used to connect in a secure way to the discovery server.
```

Comment:
Let's not confuse users with a `certificate` that's a filepath in `HandlerClient`, and a `certificate` that's a certificate content in `ApplicationClient`. If you are constrained in `ApplicationClient` to use a cert content, rename the arguments to make their content explicit (`certificate_path` in `HandlerClient`, `certificate_content` in `ApplicationClient`).